### PR TITLE
app - Selectors for the Labware Offset 

### PR DESCRIPTION
**Pull request by reachojhaneha**
I am adding ID selectors to LabwareInfoOverlay.tsx for the offset box, displayName and X,Y,Z offset values itself

- 'LabwareInfoOverlay_offset_box'
- 'LabwareInfoOverlay_displayName'
- 'LabwareInfoOverlay_X'
- 'LabwareInfoOverlay_Y'
- 'LabwareInfoOverlay_Z'
- 'LabwareInfoOverlay_X_value'
- 'LabwareInfoOverlay_Y_value'
- 'LabwareInfoOverlay_Z_value'

### CODE REVIEW

File: app/src/organisms/ProtocolSetup/RunSetupCard/LabwareSetup/LabwareInfoOverlay.tsx [hunk 1]
```diff
@@ -91,8 +91,13 @@ const LabwareInfo = (props: LabwareInfoProps): JSX.Element | null => {
       borderRadius={`0 0 0.4rem 0.4rem`}
       fontSize={FONT_SIZE_CAPTION}
       color={C_WHITE}
+      id="LabwareInfoOverlay_offset_box"
```
Comment (shlokamin) [inline app/src/organisms/ProtocolSetup/RunSetupCard/LabwareSetup/LabwareInfoOverlay.tsx]: this id is formatted differently than the one below — the second work here is kebab case and in the id below it is camel case. let's just stick to the style you have below?

`id="LabwareInfoOverlay_displayName"`

File: app/src/organisms/ProtocolSetup/RunSetupCard/LabwareSetup/LabwareInfoOverlay.tsx [hunk 2]
```diff
@@ -110,41 +115,47 @@ const LabwareInfo = (props: LabwareInfoProps): JSX.Element | null => {
...
               marginRight={'0.15rem'}
+              id="LabwareInfoOverlay_X"
             >
               X
             </Text>
             <Text
               as={'span'}
               fontWeight={FONT_WEIGHT_REGULAR}
               marginRight={'0.35rem'}
+              id="LabwareInfoOverlay_X_value"
```
Comment (shlokamin) [inline app/src/organisms/ProtocolSetup/RunSetupCard/LabwareSetup/LabwareInfoOverlay.tsx]: same this here, this is different. let's go with `ComponentName_descriptionInCamelCase`

File: app/src/organisms/ProtocolSetup/RunSetupCard/LabwareSetup/LabwareInfoOverlay.tsx [hunk 3]
```diff
@@ -110,41 +115,47 @@ const LabwareInfo = (props: LabwareInfoProps): JSX.Element | null => {
               as={'span'}
               fontWeight={FONT_WEIGHT_SEMIBOLD}
               marginRight={'0.15rem'}
+              id="LabwareInfoOverlay_X"
```
Comment (shlokamin) [inline app/src/organisms/ProtocolSetup/RunSetupCard/LabwareSetup/LabwareInfoOverlay.tsx]: same thing here

File: app/src/organisms/ProtocolSetup/RunSetupCard/LabwareSetup/LabwareInfoOverlay.tsx [hunk 4]
```diff
@@ -110,41 +115,47 @@ const LabwareInfo = (props: LabwareInfoProps): JSX.Element | null => {
...
               marginRight={'0.35rem'}
+              id="LabwareInfoOverlay_X_value"
             >
               {vector.x.toFixed(1)}
             </Text>
             <Text
               as={'span'}
               fontWeight={FONT_WEIGHT_SEMIBOLD}
               marginRight={'0.15rem'}
+              id="LabwareInfoOverlay_Y"
```
Comment (shlokamin) [inline app/src/organisms/ProtocolSetup/RunSetupCard/LabwareSetup/LabwareInfoOverlay.tsx]: same thing here

File: app/src/organisms/ProtocolSetup/RunSetupCard/LabwareSetup/LabwareInfoOverlay.tsx [hunk 5]
```diff
@@ -110,41 +115,47 @@ const LabwareInfo = (props: LabwareInfoProps): JSX.Element | null => {
...
             >
               Y
             </Text>
             <Text
               as={'span'}
               fontWeight={FONT_WEIGHT_REGULAR}
               marginRight={'0.35rem'}
+              id="LabwareInfoOverlay_Y_value"
```
Comment (shlokamin) [inline app/src/organisms/ProtocolSetup/RunSetupCard/LabwareSetup/LabwareInfoOverlay.tsx]: same thing here

File: app/src/organisms/ProtocolSetup/RunSetupCard/LabwareSetup/LabwareInfoOverlay.tsx [hunk 6]
```diff
@@ -110,41 +115,47 @@ const LabwareInfo = (props: LabwareInfoProps): JSX.Element | null => {
...
             >
               {vector.y.toFixed(1)}
             </Text>
             <Text
               as={'span'}
               fontWeight={FONT_WEIGHT_SEMIBOLD}
               marginRight={'0.15rem'}
+              id="LabwareInfoOverlay_Z"
```
Comment (shlokamin) [inline app/src/organisms/ProtocolSetup/RunSetupCard/LabwareSetup/LabwareInfoOverlay.tsx]: same thing here

File: app/src/organisms/ProtocolSetup/RunSetupCard/LabwareSetup/LabwareInfoOverlay.tsx [hunk 7]
```diff
@@ -110,41 +115,47 @@ const LabwareInfo = (props: LabwareInfoProps): JSX.Element | null => {
...
             >
               Z
             </Text>
             <Text
               as={'span'}
               fontWeight={FONT_WEIGHT_REGULAR}
               marginRight={'0.35rem'}
+              id="LabwareInfoOverlay_Z_value"
```
Comment (shlokamin) [inline app/src/organisms/ProtocolSetup/RunSetupCard/LabwareSetup/LabwareInfoOverlay.tsx]: same thing here

**Review by shlokamin — CHANGES_REQUESTED** [review]
i think we should just pick one formatting style and stick to it. it looks like elsewhere we do `id="ComponentName_descriptionInCamelCase"` which works for me

File: app/src/organisms/ProtocolSetup/RunSetupCard/LabwareSetup/LabwareInfoOverlay.tsx (hunk 1 above)
Comment (reachojhaneha) [reply app/src/organisms/ProtocolSetup/RunSetupCard/LabwareSetup/LabwareInfoOverlay.tsx]: addressed this

File: app/src/organisms/ProtocolSetup/RunSetupCard/LabwareSetup/LabwareInfoOverlay.tsx (hunk 2 above)
Comment (reachojhaneha) [reply app/src/organisms/ProtocolSetup/RunSetupCard/LabwareSetup/LabwareInfoOverlay.tsx]: done

File: app/src/organisms/ProtocolSetup/RunSetupCard/LabwareSetup/LabwareInfoOverlay.tsx (hunk 3 above)
Comment (reachojhaneha) [reply app/src/organisms/ProtocolSetup/RunSetupCard/LabwareSetup/LabwareInfoOverlay.tsx]: done

File: app/src/organisms/ProtocolSetup/RunSetupCard/LabwareSetup/LabwareInfoOverlay.tsx (hunk 4 above)
Comment (reachojhaneha) [reply app/src/organisms/ProtocolSetup/RunSetupCard/LabwareSetup/LabwareInfoOverlay.tsx]: done

File: app/src/organisms/ProtocolSetup/RunSetupCard/LabwareSetup/LabwareInfoOverlay.tsx (hunk 5 above)
Comment (reachojhaneha) [reply app/src/organisms/ProtocolSetup/RunSetupCard/LabwareSetup/LabwareInfoOverlay.tsx]: done

File: app/src/organisms/ProtocolSetup/RunSetupCard/LabwareSetup/LabwareInfoOverlay.tsx (hunk 6 above)
Comment (reachojhaneha) [reply app/src/organisms/ProtocolSetup/RunSetupCard/LabwareSetup/LabwareInfoOverlay.tsx]: done

File: app/src/organisms/ProtocolSetup/RunSetupCard/LabwareSetup/LabwareInfoOverlay.tsx (hunk 7 above)
Comment (reachojhaneha) [reply app/src/organisms/ProtocolSetup/RunSetupCard/LabwareSetup/LabwareInfoOverlay.tsx]: done

**Review by shlokamin — APPROVED** [review]
LGTM! The CI failure is an issue with the windows OS on the gh runner. Tests are passing on other operating systems so feel free to merge!

**Pull request merged**